### PR TITLE
feat: added username remember support

### LINKED ISSUE #49: Remember github username
As a user, I'd like to tell the tool my github username the first time I run the tool and have it be remembered.

It could be stored in an env file or something. I should be able to send the tbtbtc command and have a flow that opens the PR without any interruptions.

### PR DESCRIPTION
**Pull request by masterchief164**
This PR fixes #49

### CODE REVIEW

**Comment by masterchief164** [comment]
This PR creates a new `.username` file to save the user's username. There was another approach too. We could also use the username stored in the machine with the command `git config user.name`, but this might lead to errors as many users don't use the same username for git and GitHub. The final decision is on you. We could do it one way or another.

**Comment by adamjonas** [comment]
This works.

tACK

File: transcriber.py [hunk 1]
```diff
@@ -45,9 +45,14 @@ def add(
...
-    print("What is your github username?")
-    username = input()
+    if os.path.isfile(".username"):
+        with open(".username", "r") as f:
+            username = f.read()
+    else:
+        print("What is your github username?")
+        username = input()
+        with open(".username", "w") as f:
+            f.write(username)
```
Comment (adamjonas) [inline transcriber.py:55]: this file should be closed after the write.

File: transcriber.py [hunk 2]
```diff
@@ -45,9 +45,16 @@ def add(
...
+    if os.path.isfile(".username"):
+        with open(".username", "r") as f:
+            username = f.read()
+        f.close()
+    else:
+        print("What is your github username?")
+        username = input()
+        with open(".username", "w") as f:
+            f.write(username)
+        f.close()
```
Comment (adamjonas) [inline transcriber.py:57]: this needs to be inside the with block. Same on line 51

File: app/application.py [hunk 3]
```diff
@@ -226,6 +226,7 @@ def write_to_file(result, url, title, date, tags, category, speakers, video_titl
     with open(file_name_with_ext, 'a') as opf:
         opf.write(meta_data + '\n')
         opf.write(transcribed_text + '\n')
+    opf.close()
```
Comment (adamjonas) [inline app/application.py:229]: this needs to be inside the with block

**Review by adamjonas — COMMENTED** [review]
some indenting needed

**Pull request merged**